src/consensus/softforks.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2009-2015 Eric Lombrozo, The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_SOFTFORKS_H
+#define BITCOIN_SOFTFORKS_H
+
+class CBlockIndex;
+namespace Consensus {
+
+struct Params;
+namespace SoftForks {
+
+enum Rule
+{
+    BIP16,
+    BIP30,
+    BIP34,
+    BIP42,
+    BIP62,
+    BIP65,
+    BIP66,
+    BIP68,
```

Comment:
In my optinion there's no need to include future softforks here (perhaps with the exception of BIP65), they can add themselves in their own PRs if they're merged after this.